instrumentation/rocketmq-client/src/main/java/brave/rocketmq/client/TracingSendMessage.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package brave.rocketmq.client;
+
+import brave.Span;
+import brave.Tracer;
+import brave.internal.Nullable;
+import brave.messaging.MessagingRequest;
+import brave.propagation.CurrentTraceContext;
+import brave.propagation.TraceContext;
+import brave.propagation.TraceContextOrSamplingFlags;
+import brave.sampler.SamplerFunction;
+import io.opentelemetry.api.internal.StringUtils;
```

Comment:
let's avoid spurious imports especially for low value functions. There's probably something like this in other code in this repo, worst case copy/pasted.